NeoContractABI.mediawiki

```diff
@@ -0,0 +1,137 @@
+<pre>
+  NEP: <to be assigned>
+  Title: NeoContract ABI
+  Author: Erik Zhang <erik@neo.org>
+  Type: Standard
+  Status: Draft
+  Created: 2017-9-13
```

Comment:
How about using `YYYY-MM-DD` format? :) Thus changing `2017-9-13` to `2017-09-13` if this date ever were to be parsed by some program?